src/widget/appPlaceholder.ts

```diff
@@ -117,8 +117,8 @@ export class AppPlaceholder extends St.Widget {
 
             if (ShellVersionMatch('3.34')) {
                 switch (symbol) {
-                    case Clutter.Return:
-                    case Clutter.KP_Enter:
+                    case Clutter.KEY_Return:
+                    case Clutter.KEY_KP_Enter:
```

Comment:
Was also related to previous compatibility the constants has changed between 2 versions